react/features/base/connection/actions.native.js

```diff
@@ -180,13 +183,13 @@ function _connectionDisconnected(connection, message) {
  *
  * @param {JitsiConnection} connection - The JitsiConnection which was
  * established.
- * @private
  * @returns {{
  *     type: CONNECTION_ESTABLISHED,
  *     connection: JitsiConnection
  * }}
+ * @public
```

Comment:
I don't think we have rule for this. I personally think that params and returns should be together (I consider them like one group) and we can add the access modifier either after or before them.